{
  control.expectAndThrow(mock.longReturningMethod(4),EXCEPTION,2,3);
  control.replay();
  try {
    mock.longReturningMethod(4);
    fail();
  }
 catch (  RuntimeException exception) {
    assertSame(EXCEPTION,exception);
  }
  try {
    mock.longReturningMethod(4);
    fail();
  }
 catch (  RuntimeException exception) {
    assertSame(EXCEPTION,exception);
  }
  control.verify();
  try {
    mock.longReturningMethod(4);
    fail();
  }
 catch (  RuntimeException exception) {
    assertSame(EXCEPTION,exception);
  }
  control.verify();
}
